src/components/menu-bar/project-title-input.jsx

```diff
@@ -41,7 +42,10 @@ class ProjectTitleInput extends React.Component {
                 placeholder={this.props.intl.formatMessage(messages.projectTitlePlaceholder)}
                 tabIndex="0"
                 type="text"
-                value={this.props.projectTitle}
+                value={this.props.projectTitle ?
```

Comment:
I think we should keep this line as it was before. The placeholder has different text and formatting, and I'm not sure handleUpdateProjectTitle will be called below. Plus, we can already use sharedMessages.defaultProjectTitle where the title is set for a default project.